mage_ai/frontend/oracle/styles/colors/main.ts

```diff
@@ -6,11 +6,14 @@ export const BLUE_HIGHLIGHT_LIGHT = '#CFF3FF';
 export const BLUE_LIGHT = '#AABFFF';
 export const DARK_GRAY_LINES = '#B9BFCA';
 export const GRAY = '#B1B8C3';
+export const GRAY_HIGHLIGHT = '#4E4E4E';
 export const GRAY_LINES = '#D8DCE3';
 export const GREEN = '#2FCB52';
 export const GREEN_LIGHT = '#ACEABA';
 export const LIGHT = '#FBFCFD';
+export const LIGHT_PINK = '#B98D95';
```

Comment:
rename this to `PINK_LIGHT_2` or something. There are two "light pinks", so this is confusing